channeldb/channel.go

```diff
@@ -2392,53 +2397,15 @@ func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg,
 			return err
 		}
 
-		// Persist the unsigned acked updates that are not included
```

Comment:
interesting that all this can just be removed 🤔

Reply:
This is the remote giving us a revocation for something we just signed, and since we now are working with the assumption that we sign for *everything*, I believe we can now just remove this logic.  I was previously working under the assumption that not everything would be signed for, and we would have to filter updates, but that isn't the case anymore.

Reply:
Makes sense. Since this is the remote revoking one of its states, it shouldn't really impact the updates we have signed or not. (I'm just thinking out loud here)